.github/workflows/publish-package.yml

```diff
@@ -42,30 +42,24 @@ jobs:
           echo ::set-output name=ref::${REPOSITORY}:${VERSION}
           echo ::set-output name=baseref::${REPOSITORYBASE}:${VERSION}
           echo ::set-output name=crdref::${REPOSITORYCRD}:${VERSION}
-      - name: docker build ratify base
-        run: |
-          docker build -f ./httpserver/Dockerfile --label org.opencontainers.image.revision=${{ github.sha }} -t ${{ steps.prepare.outputs.baseref }} .
-      - name: docker build ratify with plugin
-        run: |
-          docker build -f ./httpserver/Dockerfile --build-arg build_cosign=true --build-arg build_sbom=true --build-arg build_licensechecker=true --build-arg build_schemavalidator=true --label org.opencontainers.image.revision=${{ github.sha }} -t ${{ steps.prepare.outputs.ref }} .
-      - name: docker build ratify-crds
-        run: |
-          docker build --build-arg KUBE_VERSION="1.25.4" --build-arg TARGETOS="linux" --build-arg TARGETARCH="amd64" -f crd.Dockerfile --label org.opencontainers.image.revision=${{ github.sha }} -t ${{ steps.prepare.outputs.crdref }} ./charts/ratify/crds
       - name: docker login
         uses: docker/login-action@465a07811f14bebb1938fbed4728c6a1ff8901fc # v2.2.0
         with:
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
-      - name: docker push ratify base
+      - name: docker build ratify-crds
         run: |
-          docker push ${{ steps.prepare.outputs.baseref }}
-      - name: docker push ratify with plugin
+          docker buildx create --use
+          docker buildx build --build-arg KUBE_VERSION="1.25.4" -f crd.Dockerfile --platform linux/amd64,linux/arm64,linux/arm/v7 --label org.opencontainers.image.revision=${{ github.sha }} -t ${{ steps.prepare.outputs.crdref }} --push ./charts/ratify/crds
```

Comment:
bit of scope but what's our strategy for the KUBE_VERSION we build with?

Reply:
As discussed, we ll log an issue to update our test strategy.